docs/html/user_guide.rst

```diff
@@ -1306,6 +1306,238 @@ issue tracker`_ if you believe that your problem has exposed a bug in pip.
 
 .. _`Using pip from your program`:
 
+Backtracking user guide information
+===================================
```

Comment:
This section is presumably temporary? Is the expectation that this PR gets updated once the final location is agreed, or are you hoping to merge this with the note that we'll move it later included?

Personally, I'd rather see it in its correct place from the start, as I think moving it (and the temporary note) will be confusing.

Reply:
Gah. Apologies, thats my mistake. That shouldn't have been included.